modin/experimental/cloud/connection.py

```diff
@@ -173,15 +180,13 @@ def __build_sshcmd(self, details: ConnectionDetails, forward_port: int = None):
         for oname, ovalue in opts:
             cmdline.extend(["-o", f"{oname}={ovalue}"])
         if forward_port:
-            cmdline.extend(
-                ["-L", f"127.0.0.1:{forward_port}:127.0.0.1:{self.rpyc_port}"]
-            )
+            cmdline.extend(["-L", f"127.0.0.1:{forward_port}:127.0.0.1:{forward_port}"])
```

Comment:
Why changed: rpyc_port -> forward_port?

Reply:
Because the logic changed to forward same port that is used in rpyc server to allow both parallel connects to an existing cluster and to allow connects to already running cluster (both cases have one or more rpyc servers already running by the time we want to spawn a new one).

Reply:
Quite a serious change that needs to be written at the top of PR.

Reply:
Another reason is to (potentially) allow testing with `-n`, and certainly to allow parallel (by multiple users) testing.
But I agree, I'll add it to the PR message.